server/server_test.go

```diff
@@ -1488,6 +1493,71 @@ var _ = Describe("Server package test", func() {
 			testURL("GET", versionURL, "", nil, http.StatusUnauthorized)
 		})
 	})
+
+	Describe("Healthcheck", func() {
+		Context("DB and sync working", func() {
```

Comment:
I'd rather used `DescribeTable` for that

Reply:
done